code_generator/GdkCodeGenerator/src/Generation/Generators/CommonGeneratorUtils.cs

```diff
@@ -11,5 +11,15 @@ public static string IndentEveryNewline(string input)
         {
             return input.Replace("\r\n", "\r\n    ");
         }
+
+        public static string IndentEveryNewline(string input, int numIndents)
+        {
```

Comment:
trollnit: can cache the number of spaces and replace only once

Reply:
This is very true